programs/github-pull-request/cli.js

```diff
@@ -12,16 +12,18 @@ const getJSON = require('../../lib/getJSON');
  * @param  {String} options.bundle
  * @param  {String} options.file
  * @param  {String} options.projectName
+ * @param  {boolean} options.failOnSizeIncrease (optional) - fail the build if the size increase more than 5%
```

Comment:
Can we change the limit increase to 10? 5 sounds a bit rough

Reply:
@raphaelboukara

Reply:
yes good idea